seL4-platforms/platforms.yml

```diff
@@ -118,6 +118,10 @@ platforms:
     platform: maaxboard
     req: [maaxboard1]
     march: armv8a
+    # The MaaxBoard in machine queue expects a binary image
+    extra_build_params:
```

Comment:
We have "settings" already in some other YAML files, so we could also use this name here also?

Reply:
Yes, good idea, that should actually work.

Reply:
Is it good now?